vaadin-combo-box-item.html

```diff
@@ -70,6 +78,15 @@
             reflectToAttribute: true
           },
 
+          /*
+           * True when item is disabled
+           */
+          disabled: {
+            type: Boolean,
+            value: false,
+            reflectToAttribute: true,
```

Comment:
Remember to document this new state which as available for styling and part of the public API